packages/website/ts/components/generate_order/asset_picker.tsx

```diff
@@ -257,6 +257,7 @@ export class AssetPicker extends React.Component<AssetPickerProps, AssetPickerSt
         };
 
         newTokenEntry.isTracked = true;
+        newTokenEntry.trackedTimestamp = Date.now();
```

Comment:
I've seen us use `moment().unix()` in these instances instead of Date.

Reply:
Yeah, let's try and always use `moment`. It's a much more robust date/time library then `Date`.